common/djangoapps/student/tests/test_views.py

```diff
@@ -573,6 +573,25 @@ def test_show_empty_dashboard_message(self):
             self.assertIn('You are not enrolled in any courses yet.', response.content)
             self.assertIn(empty_dashboard_message, response.content)
 
+    @patch('student.models.CourseEnrollment.get_dashboard_course_limit')
```

Comment:
You should put two courses in the enrollment list to ensure that the rendering is working as expected an only providing one of the two courses.  With the test as you have it now, you can't be sure that the template code is not ignoring the limit.

Reply:
@feanil I am already putting two courses in enrollment list and then rendering only 1 course. `1 results successfully populated` message will be only showed if number of rendered courses are less than total enrolled course. So can you please explain what do you mean by `you can't be sure that the template code is not ignoring the limit.`?